### PR TITLE
Test enear

### PR DESCRIPTION
**Pull request by mfornet**

### CODE REVIEW

**Comment by mfornet** [comment]
@vince0656 Take a look at this PR. I made this changes while testing the connector and deploying.

**Review by vince0656 — APPROVED** [review]

**Comment by vince0656** [comment]
@mfornet looks great - approved

File: eNear/contracts/eNear.sol [hunk 1]
```diff
@@ -44,12 +44,7 @@ contract eNear is ERC20, Bridge, AdminControlled {
         uint64 _minBlockAcceptanceHeight,
         address _admin,
         uint256 _pausedFlags
-    ) public ERC20(_tokenName, _tokenSymbol) AdminControlled(_admin, _pausedFlags) {
-        nearConnector = _nearConnector;
-        prover = _prover;
-
-        minBlockAcceptanceHeight = _minBlockAcceptanceHeight;
-
+    ) public ERC20(_tokenName, _tokenSymbol) AdminControlled(_admin, _pausedFlags) Bridge(_prover, _nearConnector, _minBlockAcceptanceHeight) {
```
Comment (sept-en) [inline eNear/contracts/eNear.sol:47]: `        uint256 _pausedFlags)
    ERC20(_tokenName, _tokenSymbol)
    Bridge(_prover, _nearConnector, _minBlockAcceptanceHeight)
    AdminControlled(_admin, _pausedFlags) 
    public
{`

Let's use the correct order of the initialization and appropriate indentation.

Comment (sept-en) [reply eNear/contracts/eNear.sol:47]: The Github doesn't show up the new lines in my comment, I meant to put every constructor on the new line & the `public` modifier should be put at the end.

**Review by sept-en — APPROVED** [review]
Good job!
Just added the comment about constructors order & minor stylystic fixes.

Btw, I really like the idea of how you managed the endpoints and private key in the hardhat config. Looks neat. I will follow this way in my configs as well :)

**Comment by mfornet** [comment]
Close this in favor of: https://github.com/vince0656/near-erc20-connector/pull/2

**Pull request merged**